### ISSUE TITLE
The definition of a loss function？

### ISSUE THREAD

**Issue by OUTSIDER-CODE** [ManLiuCoder/PSVMA#5]
Why does `unsenn` appear in the loss function even though it is not used during the training phase to mitigate bias?

**Comment by OUTSIDER-CODE** [comment]
in the formula 16

**Comment by ManLiuCoder** [comment]
Thank you for your interest in our work. 

Note that only the unseen category prototypes are utilized in our approach, aligning with the definition of zero-shot learning as outlined in reference [1]. For instance, generative-based GZSL methods require these category prototypes to synthesize visual features for unseen classes. Furthermore, in embedding-based approaches, many works, such as DVBE, TCN, and DPPN, rely on these class prototypes for score computation.
[1] Triple Verification Network for Generalized Zero-Shot Learning, TIP19
